<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Nagendra-Yoga" ignored="0" name="Validating User login" tests="5" failures="1" timestamp="2021-08-26T15:00:13 PDT" time="78.43" errors="0">
  <testcase name="TC_verifyLoginPageTitle" time="13.882" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyLoginPageMandatoryfields" time="0.211" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyForgotEmailPasswordLink" time="2.383" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_verifyCreateAccountButton" time="6.33" classname="com.DollarDays.testCases.LoginPageTest"/>
  <testcase name="TC_logintoDD" time="16.331" classname="com.DollarDays.testCases.LoginPageTest">
    <failure type="java.lang.AssertionError" message="Could not Login to Dollardays with valid credentials.  Testcase failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Could not Login to Dollardays with valid credentials.  Testcase failed expected [true] but found [false]
at com.DollarDays.testCases.LoginPageTest.TC_logintoDD(LoginPageTest.java:114)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- TC_logintoDD -->
</testsuite> <!-- Validating User login -->
